docs/user-guide/response-codes-client-error.md

```diff
@@ -0,0 +1,22 @@
+# Styx Client Error (4xx) Response Codes
+
+When a client behaves in an unexpected way toward Styx, Styx will respond with specific error codes.
+This refers only to responses generated by Styx itself, not responses being proxied from downstream.
+For a general explanation of 4xx (Server Error) codes, please see the HTTP Spec.
+
+Note that interceptor plugins can also be programmed to modify response codes, so 
+they may introduce 4xx statuses for other reasons not listed here.
+
+
+## 400 Bad Request
+
+Occurs when a request is somehow incorrectly formatted. Examples:
+
+* Unrecognised HTTP verb
+* Multiple `Host` headers.
+
```

Comment:
The word `somehow` sounds bit unprofessional. Perhaps say `Occurs when the HTTP request format does not conform to HTTP 1.1 protocol specification.` etc?

Also, add `etc` at the end of the list to emphasise that the list of possible causes is not exhaustive.